client/state/imports/site-importer/actions.js

```diff
@@ -146,15 +146,9 @@ export const importSite = ( {
 	dispatch( recordTracksEvent( 'calypso_site_importer_start_import_request', trackingParams ) );
 	dispatch( startSiteImporterImport() );
 
-	wpcom.req
-		.post( {
-			path: `/sites/${ siteId }/site-importer/import-site?${ stringify( params ) }`,
-			apiNamespace: 'wpcom/v2',
-			formData: [
-				[ 'import_status', JSON.stringify( toApi( importerStatus ) ) ],
-				[ 'site_url', targetSiteUrl ],
-			],
-		} )
+	wpcom
+		.undocumented()
+		.importWithSiteImporter( site.ID, toApi( importerStatus ), params, targetSiteUrl )
```

Comment:
Given we're still using `siteid` on lines 139 and 157, is it worth using that here to make things simpler?

Reply:
Good catch. Updated it to use `siteId` instead of `site.ID`.